package.json

```diff
@@ -19,8 +19,14 @@
     "cesium"
   ],
   "peerDependencies": {
-    "eslint": "^8.0.0",
-    "eslint-plugin-n": "^16.0.2",
-    "eslint-config-prettier": "^9.0.0"
+    "eslint": "^9.0.0",
+    "@eslint/js": "^9.0.0",
+    "eslint-config-prettier": "^9.0.0",
+    "eslint-plugin-n": "^17.0.0",
+    "globals": "15.0.0"
```

Comment:
I just noticed this is `15.0.0` instead of `^15.0.0`, was there a particular reason for this, or just a typo? If a typo I can push a fix and release a new version. This caused npm install errors in another project.

Reply:
This is a typo from a fresh `npm install`. Thank you for catching.